webapp/packages/core-navigation-tree/src/NodesManager/getNodesFromContext.ts

```diff
@@ -12,8 +12,8 @@ import { DATA_CONTEXT_NAV_NODES } from './DATA_CONTEXT_NAV_NODES';
 import type { NavNode } from './EntityTypes';
 
 export function getNodesFromContext(context: IDataContextProvider): NavNode[] {
-  const node = context.tryGet(DATA_CONTEXT_NAV_NODE);
-  const getNodes = context.tryGet(DATA_CONTEXT_NAV_NODES);
+  const node = context.get(DATA_CONTEXT_NAV_NODE);
+  const getNodes = context.get(DATA_CONTEXT_NAV_NODES);
```

Comment:
Seems like getNodes is always defined now

Reply:
type of `get` changed from `T` to `T | undefined` and equal to type of `tryGet`. so it's basically `tryGet` replaced `get`